### DISCUSSION TITLE
Como ativar o relatório de BALANCETE no odoo 14?

### DISCUSSION THREAD

**Discussion by gilsonlima10** [OCA/l10n-brazil#3083] [Q&A] [answered]
Como ativar o relatorio de BALANCETE no odoo 14?

**Comment by marcelsavegnago** [comment, marked as answer]
@gilsonlima10 pode colocar uma descrição na sua issue?

Pelo titulo eu entendi que o que você está procurando possa ser isso:
https://github.com/OCA/account-financial-reporting/tree/14.0/account_financial_report

**Comment by antoniospneto** [comment]
@gilsonlima10 para perguntas gerais desse tipo favor usar o fórum: https://github.com/OCA/l10n-brazil/discussions
@marcelsavegnago acho que dá de mover pra lá não ?

**Reply by marcelsavegnago** [reply]
Done

**Comment by antoniospneto** [comment]
@gilsonlima10 o balancete vai ser o relatório "Trial Balance" descrito ali no módulo que @marcelsavegnago indicou.

**Reply by gilsonlima10** [reply]
resolvido..grato!!
